### ISSUE TITLE
KeyError: 'count' for x5ih-idh7 from www.dallasopendata.com

### ISSUE THREAD

**Issue by simonw** [DallasMorningNews/socrata2sql#1]
Attempting to load this data: https://www.dallasopendata.com/Budget-Finance/Vendor-Payments/x5ih-idh7
```
(tmp) /tmp $ socrata2sql insert www.dallasopendata.com x5ih-idh7
WARNING:root:Requests made without an app_token will be subject to strict throttling limits.

Connecting to database
  ▶ Using default SQLite database "sqlite:///vendor_payments.sqlite".
  ▶ Query "SELECT PostGIS_version();" failed. Geometry columns will be skipped.

Setting up new table, "vendor_payments", from Socrata API fields
Traceback (most recent call last):
  File "/Users/simonw/.local/share/virtualenvs/tmp-agwWamBd/bin/socrata2sql", line 10, in <module>
    sys.exit(main())
  File "/Users/simonw/.local/share/virtualenvs/tmp-agwWamBd/lib/python3.7/site-packages/socrata2sql/cli.py", line 293, in main
    num_rows = get_row_count(client, dataset_id)
  File "/Users/simonw/.local/share/virtualenvs/tmp-agwWamBd/lib/python3.7/site-packages/socrata2sql/cli.py", line 194, in get_row_count
    return int(count[0]['count'])
KeyError: 'count'
```

Various others from www.dallasopendata.com worked fine for me.

**Comment by simonw** [comment]
It's a case sensitivity problem:
```
(socrata2sql) ~/Dropbox/Development/socrata2sql $ python -i $(which socrata2sql) insert www.dallasopendata.com x5ih-idh7
WARNING:root:Requests made without an app_token will be subject to strict throttling limits.

Connecting to database
  ▶ Using default SQLite database "sqlite:///vendor_payments.sqlite".
  ▶ Query "SELECT PostGIS_version();" failed. Geometry columns will be skipped.

Setting up new table, "vendor_payments", from Socrata API fields
Traceback (most recent call last):
  File "/Users/simonw/.local/share/virtualenvs/socrata2sql-SS0cucmt/bin/socrata2sql", line 11, in <module>
    load_entry_point('socrata2sql', 'console_scripts', 'socrata2sql')()
  File "/Users/simonw/Dropbox/Development/socrata2sql/socrata2sql/cli.py", line 293, in main
    num_rows = get_row_count(client, dataset_id)
  File "/Users/simonw/Dropbox/Development/socrata2sql/socrata2sql/cli.py", line 194, in get_row_count
    return int(count[0]['count'])
KeyError: 'count'
>>> import pdb; pdb.pm()
> /Users/simonw/Dropbox/Development/socrata2sql/socrata2sql/cli.py(194)get_row_count()
-> return int(count[0]['count'])
(Pdb) count
[{'COUNT': '161936'}]
(Pdb) 
```

**Comment by achavez** [comment]
Ahhh ... About to push up a version with an `as count` thrown into that query that ought to fix it. Thanks for the bug report.

**Comment by simonw** [comment]
I like your fix better than mine :)

**Comment by achavez** [comment]
Fixed in 0.1.2: https://pypi.org/project/socrata2sql/ Thanks again for the bug report, @simonw.
